### PR TITLE
Define and activate release v12.7.0 for AWS

### PR DESCRIPTION
**Pull request by alex-dabija**
AWS release v12.7.0 upgrades aws-cni to v1.7.5 and makes batch size and batch wait time configurable for tenant cluster upgrades.

### CODE REVIEW

File: aws/v12.7.0/release.diff [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp        giantswarm.io/docs: https://docs.giantswarm.io/reference/cp
+  creationTimestamp: null                                            creationTimestamp: null
+  name: v12.6.0                                                 |    name: v12.7.0
+spec:                                                              spec:
+  apps:                                                              apps:
+  - name: cert-exporter                                              - name: cert-exporter
+    version: 1.2.3                                                     version: 1.2.3
+  - componentVersion: 1.0.2                                          - componentVersion: 1.0.2
+    name: cert-manager                                                 name: cert-manager
+    version: 2.3.0                                                     version: 2.3.0
```
Comment (xh3b4sd) [inline aws/v12.7.0/release.diff:16]: In https://github.com/giantswarm/giantswarm/issues/14284 we got asked to update to `2.3.2`. There is also [a "request" for aws](https://github.com/giantswarm/releases/blob/ac45b2ece0ffa6ebff630358a8eb2f0c2b04b626/aws/requests.yaml#L10-L17).

Comment (calvix) [reply aws/v12.7.0/release.diff]: done

**Comment by calvix** [comment]
awscnf plans executed sucesfully

**Comment by calvix** [comment]
cncf is somehow stuck

**Comment by calvix** [comment]
now CNCF looks good, lets wait until it finishes

**Comment by calvix** [comment]
last cncf test finished without issues

**Comment by calvix** [comment]
this is ready to merge from my side

**Review by calvix — APPROVED** [review]

File: aws/v12.7.0/release.diff [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Generated with:                                               <
+# devctl release create --name 12.6.0 --provider aws --base 12. <
```
Comment (xh3b4sd) [inline aws/v12.7.0/release.diff:2]: Why is this header gone? Was that changed in the tool or was the version used old?

**Review by xh3b4sd — APPROVED** [review]

File: aws/v12.7.0/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# :zap: Giant Swarm Release v12.7.0 for AWS :zap:
```
Comment (marians) [inline aws/v12.7.0/README.md:1]:
```suggestion
# :zap: Tenant Cluster Release v12.7.0 for AWS :zap:
```

To adapt to https://github.com/giantswarm/giantswarm/issues/14339

Comment (alex-dabija) [reply aws/v12.7.0/README.md]: Good catch! I forgot about it. I updated the PR. Please, have another look.

File: aws/v12.7.0/announcement.md [hunk 4]
```diff
@@ -0,0 +1 @@
+**Giant Swarm release v12.7.0 for AWS is available**. This release offers the possibility to configure the subnet size of Network Pools, the size and wait time of batches during tenant cluster upgrades. Further details can be found in the [release notes](https://github.com/giantswarm/releases/tree/master/aws/v12.7.0).
```
Comment (marians) [inline aws/v12.7.0/announcement.md:1]:
```suggestion
**Tenant cluster release v12.7.0 for AWS is available**. This release offers the possibility to configure the subnet size of Network Pools, the size and wait time of batches during tenant cluster upgrades. Further details can be found in the [release notes](https://github.com/giantswarm/releases/tree/master/aws/v12.7.0).
```

**Review by marians — APPROVED** [review]
LGTM once the announcement wording is updated

**Pull request merged**